react-native-libraries.json

```diff
@@ -9440,5 +9440,13 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/pushpender-singh-ap/react-native-scanner",
+    "npmPkg": "@pushpendersingh/react-native-scanner",
+    "examples": ["https://github.com/pushpender-singh-ap/react-native-scanner/tree/main/example"],
+    "ios": true,
+    "android": true,
+    "newArchitecture": true
```

Comment:
Hello @pushpender-singh-ap, thank you for adding new library! 👍 

```suggestion
```

In case of your package, the `newArchitecture` flag can be removed, since **package.json** includes codegen config (flag will be automatically set).

Also, I will wait with merging this till tomorrow, since for newly published packages npm not always returns correct data, which can break our update script.